.cirrus.yml

```diff
@@ -21,3 +22,21 @@ task:
   build_script: |
     . /etc/profile.d/lmod.sh
     tests/ci/run_build.py ohpc $(tests/ci/cirrus_get_changed_files.sh)
+
+openeuler_task:
+  name: openEuler on aarch64
+  timeout_in: 120m
+  arm_container:
+    image: docker.io/openeuler/openeuler:22.03-lts
+    cpu: 4
+    memory: 12G
+  script: uname -a
+  setup_script: |
+    # TODO openEuler: Replace the urls below with http://xyz.openhpc.community once openEuler is added to the officially supported OSes
+    dnf install -y openEuler-release dnf-plugins-core git python3 make http://121.36.3.168:82/home:/huangtianhua:/ohpc/standard_aarch64/aarch64/ohpc-release-2-1.oe2203.ohpc.2.0.0.aarch64.rpm findutils rpm-build wget gawk jq which
+    dnf install -y https://github.com/ohpc-openeuler/ohpc/releases/download/2.x-openEuler-gcc12-aarch64/gnu12-compilers-ohpc-12.2.0-1.ohpc.2.0.0.aarch64.rpm
```

Comment:
This line should be removed. This is a hack way to install gcc for every package, it's not for each package, once we build gcc in ohpc obs, we should use it directly.